opensrp-chw-core/src/main/java/org/smartregister/chw/core/activity/CoreAncRegisterActivity.java

```diff
@@ -143,6 +146,18 @@ public void startFormActivity(JSONObject jsonForm) {
                 Timber.e(e);
             }
 
+            try {
+                List<CoreFamilyMemberModel> coreFamilyMemberModels = FamilyMemberDao.getMaleFamilyMembers(familyBaseEntityId);
+                JSONObject linkToFather = CoreJsonFormUtils.getFieldJSONObject(jsonArray, "link_to_father");
```

Comment:
Looks like the `link_to_father` field will be in the CRVS specific registration form? If so, the `JSONObject linkToFather` will almost always be NULL for other implementations. In that case, would it be better to skip any processing if `linkToFather` is NULL? i.e. Use an if statement here instead?

Reply:
Condition added.